client/src/elm/Utils/NominalDate.elm

```diff
@@ -94,6 +94,34 @@ renderAgeMonthsDaysParts language birthDate now =
     [ monthPart, dayPart ]
 
 
+renderAgeYearsMonths : Language -> NominalDate -> NominalDate -> String
+renderAgeYearsMonths language birthDate now =
```

Comment:
This is a generic function and it should be covering all options.
It doesn't matter that we're using it only from 2 years.

Options are:
0 / 1 /  1+ years
0 / 1 /  1+ months

Should be something like: 

```
case years of 
  0 ->
    case months of
        0 ->
          ..

        1 ->
          ..
        
        _ ->
          ..
          

  1 ->
      ..


  _ ->
      ..
```